README.rst

```diff
@@ -67,6 +67,7 @@ Examples::
     sqlacodegen postgresql:///some_local_db
     sqlacodegen --generator tables mysql+pymysql://user:password@localhost/dbname
     sqlacodegen --generator dataclasses sqlite:///database.db
+    sqlacodegen oracle+oracledb://user:pass@127.0.0.1:1521/XE --engine-arg thick_mode=True
```

Comment:
I think some mention of how the value of `--engine-arg` is parsed would be nice.

Reply:
Update noth the README and the help text